tests/unit/mock/config/compliance/compliance/citrix_netscaler/netscaler_basic_backup.txt

```diff
@@ -0,0 +1,85 @@
+add authentication tacacsAction AAA_ACT_TACACS_01 -serverIP 203.0.113.1 -authTimeout 10 -tacacsSecret abcdef1234 -authorization OFF -accounting ON -groupAttrName memberof
+add authentication tacacsAction AAA_ACT_TACACS_02 -serverIP 203.0.113.1 -authTimeout 10 -tacacsSecret abcdef1234 -authorization OFF -accounting ON -groupAttrName memberof
+add authentication Policy AAA_POL_TACACS_01 -rule true -action AAA_ACT_TACACS_01
+add authentication Policy AAA_POL_TACACS_02 -rule true -action AAA_ACT_TACACS_02
+bind system global AAA_POL_TACACS_01 -priority 10 -gotoPriorityExpression NEXT
+bind system global AAA_POL_TACACS_02 -priority 20 -gotoPriorityExpression NEXT
+add system group Admin -timeout 900
+bind system group Admin -policyName superuser 100
+add system group Support -timeout 900
+bind system group Support -policyName XX-CMD-read-only 100
+bind system group Support -policyName XX-CMD-partition-read-only 110
+add system group Networking -timeout 900
+bind system group Networking -policyName XX-CMD-operator 100
+bind system group Networking -policyName XX-CMD-partition-operator 110
+add system cmdPolicy XX-CMD-read-only ALLOW (^man.*)|(^show\s+(\?!system)(\?!configstatus)(\?!audit messages)(\?!techsupport).*)|(^stat.*)
+add system cmdPolicy XX-CMD-operator ALLOW (^show.*)|(^stat.*)|(^(enable|disable) (server|service).*)
+add system cmdPolicy XX-CMD-partition-read-only ALLOW (^man.*)|(^switch)|(^show\s+(\?!system)(\?!configstatus)(\?!audit messages)(\?!techsupport).*)|(^stat.*)
+add system cmdPolicy XX-CMD-partition-operator ALLOW (^man.*)|(^switch)|(^show\s+(\?!system)(\?!configstatus)(\?!audit messages)(\?!techsupport).*)|(^stat.*)|(^(enable|disable) (server|service).*)
+set audit syslogParams -userDefinedAuditlog YES
+set audit nslogParams -userDefinedAuditlog YES
+add audit syslogAction sys_act_fdi_rsyslog 203.0.113.1 -logLevel EMERGENCY ALERT CRITICAL ERROR WARNING NOTICE INFORMATIONAL -timeZone LOCAL_TIME -userDefinedAuditlog YES -transport UDP
+add audit syslogPolicy sys_pol_fdi true sys_act_fdi_rsyslog
+bind audit syslogGlobal -policyName sys_pol_fdi -priority 2000000010
+set snmp alarm CPU-USAGE -thresholdValue 95 -normalValue 35 -severity Informational
+set snmp alarm HA-STATE-CHANGE -severity Informational
+set snmp alarm IP-CONFLICT -severity Warning
+set snmp alarm MEMORY -thresholdValue 95 -normalValue 35 -severity Critical
+set snmp alarm POWER-SUPPLY-FAILURE -severity Minor
+set snmp alarm SSL-CARD-FAILED -severity Minor
+set snmp alarm SSL-CERT-EXPIRY -severity Warning
+add snmp view READ 1 -type included
+add snmp group NETMON-GROUP authpriv -readViewName READ
+add snmp user monitoring -group NETMON-GROUP -authType SHA -authpasswd abcdef1234 -privType AES -privpasswd abcdef1234
+add ssl cipher XX-CIPHER-GROUP_1.0_v01
+bind ssl cipher XX-CIPHER-GROUP_1.0_v01 -cipherName TLS1.2-ECDHE-RSA-AES256-GCM-SHA384
```

Comment:
@jeffkala mentioned, and just want to confirm, would we ever want a parent and child relationship for add & bind. I don't think so, but just want to think of it.

Reply:
@jdrew82 think I was thinking specifically like this cipher group `XX-CIPHER-GROUP_1.0_v01` the add accomplishes only a subset of the configuration.  Just wondering if the `add` line should be the parent and the `bind` lines would be children since they appear to be related.

Reply:
I think it might be difficult to make add/bind as a global parent/child relationship when it comes to NetScaler. If I remember correctly, the `add` line is done to create an object and the `bind` line is to apply it to something else typically. In this specific case, it's creating a cipher group and adding ciphers into it by the looks of it. However, when it comes to something like a VIP you'd be binding the profile to the virtual server.

Reply:
I am good with it.